frontend/src/schemas/SchemaList.tsx

```diff
@@ -79,6 +145,22 @@ export default function SchemaList({ schemaKind }: SchemaDisplayProps) {
     return <MessageAlert message={isSchemasError.info.message} severity='error' />
   }
 
+  if (isModelsError) {
+    return <MessageAlert message={isModelsError.info.message} severity='error' />
+  }
+
+  if (isModelsLoading) {
```

Comment:
We can combine all loading conditions into one and they should be below all error blocks, so below `isModelsError` and `isReviewsError` in this case:
```
  if (isModelsLoading || isReviewsLoading) {
    return <Loading />
  }
```